kea_exporter/kea.py

```diff
@@ -236,11 +274,16 @@ def setup_dhcp4_metrics(self):
             },
             'total-addresses': {
                 'metric': 'addresses_total',
+            },
+            'v4-reservation-conflicts' :{
+                'metric' : 'reservation_conflicts',
```

Comment:
This is a unit-less accumulation and should therefore use the `_total` suffix.

```suggestion
                'metric' : 'reservation_conflicts_total',
```